From e339f0e2ec853f413f881687bb9790a852fbdd1c Mon Sep 17 00:00:00 2001
Message-Id: <e339f0e2ec853f413f881687bb9790a852fbdd1c.1602412141.git-series.worldofpeace@protonmail.ch>
In-Reply-To: <933ae4eecb696213bcd8aa881ba33ed60544fbfd.1602412141.git-series.worldofpeace@protonmail.ch>
References: <933ae4eecb696213bcd8aa881ba33ed60544fbfd.1602412141.git-series.worldofpeace@protonmail.ch>
From: WORLDofPEACE <worldofpeace@protonmail.ch>
Date: Sun,  5 Apr 2020 23:06:03 -0400
Subject: [PATCH 3/8] Fix glitches in gala

From: worldofpeace <worldofpeace@protonmail.ch>

This fixes issues for users of mutter like in gala[0].

Upstream report: https://gitlab.gnome.org/GNOME/mutter/issues/536
[0]: https://github.com/elementary/gala/issues/605
---
 clutter/clutter/clutter-actor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/clutter/clutter/clutter-actor.c b/clutter/clutter/clutter-actor.c
index ecf9a59..07b8b71 100644
--- a/clutter/clutter/clutter-actor.c
+++ b/clutter/clutter/clutter-actor.c
@@ -17831,7 +17831,7 @@ _clutter_actor_get_paint_volume_mutable (ClutterActor *self)
   if (_clutter_actor_get_paint_volume_real (self, &priv->paint_volume))
     {
       priv->paint_volume_valid = TRUE;
-      priv->needs_paint_volume_update = FALSE;
+      //priv->needs_paint_volume_update = FALSE;
       return &priv->paint_volume;
     }
   else
-- 
git-series 0.9.1
